### PR TITLE
CB-16677 Extend Cloudbreak README with Consumption service setup

### PR DESCRIPTION
**Pull request by nadamdb**
See detailed description in the commit message.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -366,6 +367,27 @@ AWS_SECRET_ACCESS_KEY=
 CB_AWS_ACCOUNT_ID=
 ```
 
+### Running the Consumption service in IDEA
+
+After importing the `cloudbreak` repo root, launch the Consumption application by executing the `com.sequenceiq.consumption.ConsumptionApplication` class (set `Use classpath of module` to `cloudbreak.cloud-consumption.main`) with the following JVM options:
```
Comment (doktoric) [inline README.md:372]: can you run the application from cbd as well?

Comment (SubhanHakverdiyev) [reply README.md:372]: you mean by adding it to the CB_LOCAL_DEV_LIST?

Comment (doktoric) [reply README.md:372]: yes

Comment (SubhanHakverdiyev) [reply README.md:372]: yes if you remove it the consumption container will  run by cbd command.

**Comment by SubhanHakverdiyev** [comment]
retest this please

**Comment by SubhanHakverdiyev** [comment]
retest this please

**Pull request merged**